test/powershell/Modules/Microsoft.PowerShell.Utility/Select-String.Tests.ps1

```diff
@@ -175,6 +184,24 @@ Describe "Select-String" -Tags "CI" {
 
 	    Select-String 'matc*' $testInputFile -ca | Should -Match $expected
 	}
+
+	It "Should return all strings where 'in' is found in testfile1, when -Raw is used." {
+	    $expected1 = "This is a text string, and another string"
+	    $expected2 = "This is the second line"
+	    $expected3 = "This is the third line"
+	    $expected4 = "This is the fourth line"
+
+	    (Select-String in $testInputFile -Raw)[0] | Should -Be $expected1
```

Comment:
Please use BeExactly for strings. Below too.
```suggestion
	    (Select-String in $testInputFile -Raw)[0] | Should -BeExactly $expected1
```

Reply:
I don't think that is the right thing to do in this case: Select-String is case insensitive, unless `-CaseSensitive` is specified. Are you sure?

Reply:
We do not check "case insensitive". The check should guarantee that we received _exactly_ the original string. Perhaps it is not critical for the test but it is best practice.

Reply:
OK. I see your point, and I agree now. Would you like me to retrofit this same fix to lines 133-137?

Reply:
You can do this in the PR (or move to follow PR).